pkgs/data/misc/v2ray-geoip/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```

Comment:
Could be simpler:
```
let
  version = "202111250029";
in
fetchurl {
  name = "v2ray-geoip-${version}";
  url = ""https://github.com/v2fly/geoip/releases/download/${version}/geoip.dat";
  sha256 = ...

  downloadToTemp = true;
  recursiveHash = true;

  postFetch = "install -Dm0644 $downloadedFile $out/share/v2ray/geoip.dat";
  meta = ...
}
```

Reply:
However this approach makes it hard to do things like `nix-prefetch-url`

Reply:
Why? `nix-prefetch-url https://github.com/v2fly/geoip/releases/download/202111250029/geoip.dat`

Reply:
`recursiveHash = true;` is for the output hash, not the input.